src/Chapter3.hs

```diff
@@ -907,9 +1017,27 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+
+newtype Gold   = MkGold   { unGold   :: Int } deriving (Show)
+
 class Append a where
     append :: a -> a -> a
 
+instance Append Gold where
+    append :: Gold  -> Gold -> Gold
+    append a b = MkGold (unGold a + unGold b)
+
+instance Append [a] where
+    append :: [a]  -> [a] -> [a]
+    append l1 l2 = l1 ++ l2
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
    append = (++)
```